sync/config.py

```diff
@@ -141,12 +168,26 @@ def get_databricks_config() -> DatabricksConf:
     return _db_config
 
 
+def get_profile_name() -> str:
+    """Gets the active profile
+
+    :return: active profile
+    :rtype: str
+    """
+    global _active_profile
+    _active_profile = _current_profile_dir().resolve().name
```

Comment:
I'm not seeing where we depend on the gobal `_active_profile` being correctly set. I we don't depend on it then we're probably better off without it - won't have to worry about whether it correctly represents the state of the symlink.